op-bindings/predeploys/addresses.go

```diff
@@ -0,0 +1,17 @@
+package predeploys
+
+import "github.com/ethereum/go-ethereum/common"
+
+var (
```

Comment:
Since these are globals, it would be good to have them as string `const`s imo

Reply:
Callers can instantiate `common.Address`

Reply:
TBH, any chance to do both (i.e. have const string and address versions). We can also use address literals here - slightly more work, but still const.

Reply:
Something like this `var WithdrawalContractAddress2 = common.Address{0x42, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 16}`
Unfortunately you can't make it const. As such, const strings + a set of `common.HexToAddresses` (to avoid the conversion is probably what makes sense)